private/model/api/docstring_test.go

```diff
@@ -0,0 +1,52 @@
+package api
```

Comment:
The PR #891 will also fix this too for users in general. but i think both are needed for these functions because they use 1.6+ features.